frontend/integration-tests/tests/storage.scenario.ts

```diff
@@ -0,0 +1,55 @@
+import { browser, ExpectedConditions as until, by, element } from 'protractor';
+import { appHost, testName, checkLogs, checkErrors } from '../protractor.conf';
+import * as crudView from '../views/crud.view';
+import { execSync } from 'child_process';
+import { OrderedMap } from 'immutable';
+import * as yamlView from '../views/yaml.view';
+
+describe('Add storage is applicable for all workloads', () => {
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  const k8sObjs = OrderedMap<string, {kind: string, namespaced?: boolean}>()
+    .set('replicationcontrollers', {kind: 'ReplicationController'})
+    .set('daemonsets', {kind: 'DaemonSet'})
+    .set('deployments', {kind: 'Deployment'})
+    .set('replicasets', {kind: 'ReplicaSet'})
+    .set('statefulsets', {kind: 'StatefulSet'});
+
+  const openshiftObjs = OrderedMap<string, {kind: string, namespaced?: boolean}>()
+    .set('deploymentconfigs', {kind: 'DeploymentConfig'});
+  const resourceObjs = browser.params.openshift === 'true' ? k8sObjs.merge(openshiftObjs) : k8sObjs;
+
+  resourceObjs.forEach(({kind}, resource) => {
+    describe(kind, () => {
+      it('create a new resource', async() => {
```

Comment:
This seems to have a lot of common code with the curd scenario. It would be better to put this in a common helper somewhere instead of copying it here.

Reply:
Yes, make sense,